ethereum/smart-contract/contracts/interfaces/IGatewayNetwork.sol

```diff
@@ -24,9 +25,9 @@ abstract contract  IGatewayNetwork {
         // Unique identifier for a network.
         bytes32 name;
         
-        // The default expiration timestamp of passes on this network.
+        // The default expiration time in seconds of passes on this network.
         // A value of 0 means passes never expire
-        uint256 passExpireTimestamp;
+        uint256 passExpireTimeInSeconds;
```

Comment:
Could we call this passExpireDurationInSeconds?

Reply:
Yep that name is a bit clearer. Updated